pkg/sfu/buffer/videolayerutils.go

```diff
@@ -0,0 +1,306 @@
+package buffer
+
+import (
+	"github.com/livekit/protocol/livekit"
+	"github.com/livekit/protocol/logger"
+)
+
+const (
+	QuarterResolution = "q"
+	HalfResolution    = "h"
+	FullResolution    = "f"
+)
+
+func LayerPresenceFromTrackInfo(trackInfo *livekit.TrackInfo) map[livekit.VideoQuality]bool {
```

Comment:
maybe better to use a slice or byte than map

Reply:
Sounds good @cnderrauber . Guess I can just make three bools :-). Will make it a slice.